src/Language/PureScript/Errors.hs

```diff
@@ -687,6 +691,12 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
       paras [ line "An exhaustivity check was abandoned due to too many possible cases."
             , line "You may want to decomposing your data types into smaller types."
             ]
+    renderSimpleErrorMessage (UnusedImport name) =
+      line $ "The import of module " ++ runModuleName name ++ " is redundant"
+
+    renderSimpleErrorMessage (UnusedExplicitImport name names) =
+      paras [ line $ "The import of module " ++ runModuleName name ++ " contains the following redundant references:"
```

Comment:
Perhaps we could phrase this as "contains the following unused references" instead?